javascript-view-swapping/main.js

```diff
@@ -0,0 +1,30 @@
+var $tabContainer = document.querySelector('.tab-container');
+var $tabList = document.querySelectorAll('.tab');
+var $viewList = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', tabSwitch);
+
+function tabSwitch() {
+
+  if (event.target.matches('.tab')) {
+    for (var i = 0; i < $tabList.length; i++) {
+      if ($tabList[i] === event.target) {
+        $tabList[i].className = 'tab active';
+      } else {
+        $tabList[i].className = 'tab';
+      }
+    }
+  }
+
+  if (event.target) {
```

Comment:
What happens if you click something that's not a tab, like the area next to the tabs? Does everything disappear?